tests/integration/actions/test_inventory.py

```diff
@@ -0,0 +1,86 @@
+""" inventory integration tests
+"""
+import json
+import os
+import pytest
+
+from .._common import get_executable_path
+from .._common import TmuxSession
+
+fixtures_dir = os.path.join(os.path.dirname(__file__), "..", "..", "fixtures")
+
+CLI = (
+    f"{get_executable_path('python')}"
+    " -m ansible_navigator inventory"
+    f" -i {os.path.join(fixtures_dir,'inventory')}"
+)
+
+testdata = [
+    (0, CLI, "ansible-navigator inventory command top window"),
```

Comment:
Just a note, this approach is dependant on the use of : commands sine we look for :help after a command to determine the screen has been updated

Reply:
+1